dbt/include/bigquery/macros/materializations/incremental_strategy/insert_overwrite.sql

```diff
@@ -1,5 +1,5 @@
 {% macro bq_generate_incremental_insert_overwrite_build_sql(
-    tmp_relation, target_relation, sql, unique_key, partition_by, partitions, dest_columns, on_schema_change, copy_partitions
+    tmp_relation, target_relation, sql, unique_key, partition_by, partitions, dest_columns, tmp_relation_exists, copy_partitions
```

Comment:
[Align the parameter naming on the call site variable name](https://github.com/Kayrnt/dbt-bigquery/blob/5b691e3b848cda102ba8f0688a4ab850475bf3c2/dbt/include/bigquery/macros/materializations/incremental.sql#L48)